bears/vcs/git/GitCommitBear.py

```diff
@@ -34,14 +34,22 @@ class GitCommitBear(GlobalBear):
             'issue': r'(?:\w+/\w+)?#(\S+)',
             'full issue': r'https?://gitlab\S+/issues/(\S+)',
         },
+        'bitbucket': {
+            'issue': r'#(\S+)',
+            'full issue': None,
+        },
     }
     SUPPORTED_HOST_KEYWORD_REGEX = {
         'github': (r'[Cc]lose[sd]?'
                    r'|[Rr]esolve[sd]?'
                    r'|[Ff]ix(?:e[sd])?'),
         'gitlab': (r'[Cc]los(?:e[sd]?|ing)'
                    r'|[Rr]esolv(?:e[sd]?|ing)'
-                   r'|[Ff]ix(?:e[sd]|ing)?')
+                   r'|[Ff]ix(?:e[sd]|ing)?'),
+        'bitbucket': (r'[Cc]los(?:e[sd]?|ing)'
+                      r'|[Rr]esolv(?:e[sd]?|ing)'
+                      r'|[Ff]ix(?:e[sd]|ing)?'
+                      r')(?: ?(?:bug|issue|ticket)?'),
```

Comment:
Replace the whitespace  with \s, this is because any kind of whitespace is supported during regex joining.

Reply:
The whitespace shouldn't be optional either if something is to proceed after the reference type. The regex should rather group the space and descriptor and make it optional (https://regex101.com/r/gJ9oKe/1/).

Reply:
Thanks @ksdme 

But I am not sure if we want to match with a newline or carriage return (since a ```\n``` or ```\r``` do not result in closing of the issue

Though yes thanks for pointing this out, multiple spaces and tabs do need to be accomodated :sweat_smile:

Reply:
Updated @ksdme 
Please review :)